### PR TITLE
Presentation 2

### PR DESCRIPTION
**Pull request by jaresan**
Basic datepicker.
Ambassador form validation.
Animal detail info taken from JSON (only panda for now).

### CODE REVIEW

File: src/components/DatePicker.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react';
```
Comment (jlelic) [inline src/components/DatePicker.js]: We should probably agree on file naming conventions, all other components are currently kebab-case

File: src/components/DatePicker.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    date[part] = value;
+
+    this.setState({ date });
+    this.props.onChange(date);
+  }
+
+  getMonthSelect() {
+    const months = [
+      'January',
+      'February',
```
Comment (jlelic) [inline src/components/DatePicker.js]: `moment.months()`

Comment (jaresan) [reply src/components/DatePicker.js]: Is he gonna check the code? 🤔

File: src/components/DatePicker.js (hunk 1 above)
Comment (jaresan) [reply src/components/DatePicker.js]: React is usually camelCase, but don't think it matters. Changed to kebab 🍕

File: src/components/DatePicker.js (hunk 2 above)
Comment (jaresan) [reply src/components/DatePicker.js]: :+1:

**Pull request merged**